lib/generators/module_component/templates/app/controllers/controller.rb.erb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module <%= class_name %>
+  module V0
+    class <%= class_name %>Controller < ApplicationController
+      skip_before_action :authenticate
```

Comment:
Should we blanket skip authentication on module generation? Seems like the default would be to include rather than exclude.

Reply:
Yeah i'm not sure, but I think i'll remove this line. I pulled this template file from the original implementation